insights/client/archive.py

```diff
@@ -250,7 +253,7 @@ def cleanup_previous_archive(self):
         '''
         Used at the start, this will clean the temporary directory of previous killed runs
         '''
-        for file in glob.glob('/var/tmp/insights-archive-*'):
+        for file in glob.glob(os.path.join(constants.insights_tmp_path, 'insights-archive-*')):
```

Comment:
This expression seems to me too complex. I suggest assigning the inner call result to an intermediary variable.

```suggestion
        archive_glob = os.path.join(constants.insights_tmp_path, 'insights-archive-*')
        for file in glob.glob(archive_glob):
```